ImageProcessing/Re-RunHSC.ipynb

```diff
@@ -0,0 +1,442 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# HSC Re-Run: Making Forced Photometry Light Curves from Scratch\n",
+    "\n",
+    "<br>Owner: **Justin Myles** ([@jtmyles](https://github.com/LSSTScienceCollaborations/StackClub/issues/new?body=@jtmyles))\n",
+    "<br>Last Verified to Run: **2018-08-10**\n",
+    "<br>Verified Stack Release: **16.0**\n",
+    "\n",
+    "This project addresses issue [#63: HSC Re-run](https://github.com/LSSTScienceCollaborations/StackClub/issues/63)\n",
+    "\n",
+    "This notebook demonstrates the [LSST Science Piplines data processing tutorial](https://pipelines.lsst.io/) with emphasis on how a given [obs package](https://github.com/lsst/obs_base) works under the hood of the command line tasks. It makes use of the [obs_subaru](https://github.com/lsst/obs_subaru) package to measure a forced photometry light curve for a small patch in the HSC sky in the [ci_hsc](https://github.com/lsst/ci_hsc/) repository. \n",
+    "\n",
+    "### Learning Objectives:\n",
+    "After working through and studying this notebook you should be able to understand how to use the DRP pipeline from image visualization through to a forced photometry light curve. Specific learning objectives include: \n",
+    "   1. [configure command-line tasks](https://pipelines.lsst.io/v/w-2018-12/modules/lsst.pipe.base/command-line-task-config-howto.html) for your science case\n",
+    "   2. TODO\n",
+    "   \n",
+    "Other techniques that are demonstrated, but not empasized, in this notebook are\n",
+    "   1. Use the `butler` to fetch data\n",
+    "   2. Visualize data with the LSST Stack\n",
+    "   3. TODO\n",
+    "\n",
+    "### Logistics\n",
+    "This notebook is intended to be runnable on `lsst-lspdev.ncsa.illinois.edu` from a local git clone of https://github.com/LSSTScienceCollaborations/StackClub.\n",
+    "\n",
+    "\n",
+    "## Set Up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.system(\"eups list lsst_distrib\") #todo\n",
+    "import sys\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "import eups.setupcmd"
+   ]
+  },
+  {
```

Comment:
Now that you have your bash script that exercises all the tasks in the pipeline, how should we make the notebook relate to it? One possibility could be to have a code cell that contains
```
! cat Re-RunHSC.sh
```
so that the script is printed into the notebook outputs when it is run, for the user to look at (and refer back to). Then, later on we could think of re-printing snippets from the bash script, using s simple script-parsing `def`.  Your script has some nice markers in it that could enable that (eg the `echo` commands, the section comments, the `date`  commands, etc). Don't worry about this though, the `cat` cell should be good enough for now.

Reply:
I implemented this, see what you think in #115 !

Reply:
Looks great, thanks!